src/components/Missions/Missions.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Mission from './Mission';
+import { loadMissions } from '../../redux/missions/missions';
+import './Missions.css';
+
+const Missions = () => {
+  const { mission } = useSelector((state) => state.missions);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(loadMissions());
+  }, []);
```

Comment:
- [ ] Kindly make sure data should not be added to the store on every re-render (i.e. when changing views / using navigation). I highly recommend to only fetch the data again only I the mission is empty because at this moment the data is fetched whenever we navigate to another view.

_**Suggestion**_
` useEffect(() => {
    if (mission.length === 0) {
      dispatch(loadMissions());
    }
  }, []);
`